Source/API/README.md

```diff
@@ -0,0 +1,20 @@
+# Auctionator External API
+
+Calling any other functions related to Auctionator is not supported and may
+break without warning in future releases.
+
+This API is intended to remain stable.
+
+```lua
+-- Returns the last scanned price for an item identified by itemID
+-- Returns the price in coppers, or nil if one wasn't found
+Auctionator.API.v1.GetAuctionPriceByItemID(callerID, itemID)
```

Comment:
We should document the parameters as well, and provide a sample call with sample results, for clarity.